clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/CramWebService.scala

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.clio.server.webservice
+
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server._
+import org.broadinstitute.clio.server.service.IndexService
+import org.broadinstitute.clio.transfer.model.BackCompatibleCramIndex
+import org.broadinstitute.clio.transfer.model.wgscram.CramKey
+import org.broadinstitute.clio.util.model.{DataType, Location}
+
+class CramWebService(cramService: IndexService[BackCompatibleCramIndex])
+    extends IndexWebService(cramService) {
+
+  private[webservice] val pathPrefixKey: Directive1[CramKey] = {
+    for {
+      location <- pathPrefix(Location.namesToValuesMap)
+      project <- pathPrefix(Segment)
+      sampleAlias <- pathPrefix(Segment)
+      version <- pathPrefix(IntNumber)
+      dataType <- pathPrefix(DataType.namesToValuesMap)
```

Comment:
Bike-shedding: since we're breaking all the IDs anyways, would it make sense to put data-type before version in all the APIs where order matters? Up until now I think we've been trying to specify segments in decreasing order of specificity.